modules/backend/formwidgets/recordfinder/partials/_recordfinder.htm

```diff
@@ -4,7 +4,9 @@
     data-control="recordfinder"
     data-refresh-handler="<?= $this->getEventHandler('onRefresh') ?>"
     data-data-locker="#<?= $field->getId() ?>">
-    <span class="form-control">
+    <span class="form-control"
+        <?= $this->previewMode ? 'disabled="disabled"' : '' ?>>
```

Comment:
One final minor style point, could you please make this and https://github.com/octobercms/october/pull/3163/files#diff-235ff395d0cbbf6067a5b0b339ce153fR9 inline with the class attribute as well instead of on new lines?

Reply:
Of course :)